Plugin/Checkout/DefaultConfigProviderPlugin.php

```diff
@@ -80,14 +93,19 @@ public function afterGetConfig($subject, $result)
             }
         }
         if ($email && (!$result['isCustomerLoggedIn'] || $yotpoAbandonedQuoteId)) {
-            $shippingAddressFromData = $this->getAddressFromData($quote->getShippingAddress());
-            $billingAddressFromData = $this->getAddressFromData($quote->getBillingAddress());
+            $shippingAddressFromData = $this->getAddressFromData($shippingAddress);
+            $billingAddressFromData = $this->getAddressFromData($billingAddress);
             $result['shippingAddressFromData'] = $shippingAddressFromData;
             if ($shippingAddressFromData != $billingAddressFromData) {
                 $result['billingAddressFromData'] = $billingAddressFromData;
             }
             $result['validatedEmailValue'] = $email;
         }
+
+        if ($billingAddress && $billingAddress->getCountryId()) {
```

Comment:
do we really need to check if `billingAddress` has a country code? because country is a mandatory field when creating a customer/checkout in Magneto. maybe this check can be removed.

Reply:
I'm actually not sure.
I'll send an email to Manaf about why it was there in the first place and delete it if we can.
WDYT?